D1673/P1673.md

```diff
@@ -6568,10 +6525,6 @@ where
         then `add_const_t<ElementType>`;
 
     * [2.1.2]{.pnum} else, `ElementType`;
-        <i>[Note:</i> For general accessors,
```

Comment:
We should _also_ remove this node because of feedback that we shouldn't try to const-ify the mdspan's `element_type`.  (It's not unreasonable to have a nonconst transposed view.  We might even want to consider removing the requirement that the result of `transposed` is read only.)